Kernel/Graphics/VirtIOGPU/GraphicsAdapter.h

```diff
@@ -18,11 +18,13 @@ class GraphicsAdapter final
     AK_MAKE_ETERNAL
 
 public:
-    static NonnullRefPtr<GraphicsAdapter> initialize(PCI::Address);
+    static RefPtr<GraphicsAdapter> initialize(PCI::Address);
```

Comment:
Can we rename this to `try_initialize` to signify it can fail? (similarly to `try_create`)